ciphers/beaufort-cipher.py

```diff
@@ -0,0 +1,57 @@
+import sys
+
+alphabet = ".ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"
+dictionary = {v: k for k, v in enumerate(alphabet)}
+
+
+def translateCharacter(char, key):
+    """Translates a character using the key and tabula recta"""
+    for x in range(0, len(alphabet)):
+        if dictionary[char] + x > len(alphabet) - 1:
+            if dictionary[char] + x - len(alphabet) == dictionary[key]:
+                return alphabet[x]
+        else:
+            if dictionary[char] + x == dictionary[key]:
+                return alphabet[x]
+
+
+def filterInput(string):
+    """Filters input from characters that aren't in the alphabet"""
+    for n in range(len(string)):
+        if not string[n] in alphabet:
+            string = string.replace(string[n], alphabet[0])
+    return string
+
+
+def assignKeyMask(input, key):
```

Comment:
__input__ is a builtin function in Python 3 so it is not a good idea to shadow it by reusing that as a variable name.

Reply:
Changed. Now passing an length in instead of full text. Name changed